provision-team/setup.sh

```diff
@@ -246,14 +246,17 @@ bash ./build_deploy_user.sh -s ./test_fetch_build -b Release -r $resourceGroupTe
 echo "10-Build and deploy Trip API to AKS  (# bash ./build_deploy_trip.sh -s ./test_fetch_build -b Release -r $resourceGroupTeam -t 'api-trip' -d $dnsURL -n ${teamName}${teamNumber} -g $registryName)"
 bash ./build_deploy_trip.sh -s ./test_fetch_build -b Release -r $resourceGroupTeam -t 'api-trip' -d $dnsURL -n ${teamName}${teamNumber} -g $registryName
 
-echo "11-Build and deploy the simulator (# bash Usage: build_deploy_simulator.sh -n ${teamName}${teamNumber} -q 18000 -t <image tag optional>)"
+echo "11-Build and User-Profile Trip API to AKS  (# bash ./build_deploy_user-profile.sh -s ./test_fetch_build -b Release -r $resourceGroupTeam -t 'api-user-java' -d $dnsURL -n ${teamName}${teamNumber} -g $registryName)"
+bash ./build_deploy_user-profile.sh -s ./test_fetch_build -b Release -r $resourceGroupTeam -t 'api-trip' -d $dnsURL -n ${teamName}${teamNumber} -g $registryName
```

Comment:
You still have:

1. `echo` ... `TRIP API`
1. `-t 'api-trip'`  Looking at the [helm chart the image tag name](https://github.com/Azure-Samples/openhack-devops-team/blob/c3a7ba6a4b43b3a5522a69d84773d0ce87c82f45/apis/user-java/helm/values.yaml#L11) needs to match what is deployed, which is `user-java`.  So it needs to be changed to this.

Reply:
oops